### ISSUE TITLE
RN 0.56 - bundle error "A trailing comma is not permitted after the rest element"

### ISSUE THREAD

**Issue by michaltaberski** [akveo/react-native-ui-kitten#127]
### Issue type

* [X] bug report
* [ ] feature request

### Issue description

Right after init plain, fresh react-native app (RN version 0.56) and adding  `react-native-ui-kitten` I am getting following error:

`/node_modules/react-native-ui-kitten/src/components/image/rkModalImg.js: A trailing comma is not permitted after the rest element (277:17)`

```
  275 |       index,
  276 |       style: imgStyle,
> 277 |       ...imgProps,
      |                  ^
  278 |     } = this.props;
```

**Comment by grass0916** [comment]
Dear @artyorsh ,

Is this issue fixed? I encounter the same problem and it cause compilation fail.

![screenshot_20180818-124543](https://user-images.githubusercontent.com/6274807/44295889-f31ff080-a2e4-11e8-8c7d-47fa11cd6766.png)

### Diff

I have removed the comma in file `src/components/image/rkModalImag.js` can work successfully.

```
276 |         style: imgStyle,
277 | -       ...imgProps,
277 | +       ...imgProps
278 |     } = this.props;
```

### Version of Kitten

```shell
$ yarn info v1.5.1
{ "name": "react-native-ui-kitten",
  "description": "React Native components",
  "dist-tags":
   { "latest": "3.0.1" },
   ...
}
```

Thanks.

**Comment by grass0916** [comment]
I had seen the others related issues.

Hope the new version will release soon.

Thanks :)

**Comment by ranjeethpt** [comment]
Same here +1

**Comment by prasadpilla** [comment]
I am waiting for the new version as well!

**Comment by usfslk** [comment]
You have to delete that comma next to imgProps but everytime you rebuild it's gonna happen again.

**Comment by prasadpilla** [comment]
@usfslk yes, that's why I am waiting for the fix. Delayed the upgrade for now

**Comment by usfslk** [comment]
> @usfslk yes, that's why I am waiting for the fix. Delayed the upgrade for now

Why does it take so much time to remove a comma and push a commit? I'm new to these report issues stuff, there's also many repos that break my code just by installing and importing them
